docs/generated/packages/next/documents/next-config-setup.md

```diff
@@ -1,132 +0,0 @@
----
-title: How to configure Next.js plugins
-description: A guide for configuring Next.js plugins with Nx
----
-
-# Configuring Next.js plugins
```

Comment:
Something is weird about this, I cannot find the moved next-config-setup in the docs/shared/... where it should be. 
![image](https://github.com/nrwl/nx/assets/542458/d675c39f-9728-43ae-9d41-4927fc9716de)

But it renders properly: https://nx-dev-git-fork-isaacplmann-docs-webpack-recipes-nrwl.vercel.app/recipes/next/next-config-setup

Reply:
docs/shared/packages/next/next-config-setup.md

Reply:
That document was unmodified by this PR.